cmd/kubeseal/main.go

```diff
@@ -58,6 +66,8 @@ func init() {
 	clientcmd.BindOverrideFlags(&overrides, flag.CommandLine, kflags)
 	clientConfig = clientcmd.NewInteractiveDeferredLoadingClientConfig(loadingRules, &overrides, os.Stdin)
 
+	pflagenv.SetFlagsFromEnv(flagEnvPrefix, flag.CommandLine)
```

Comment:
Could you re-implement `github.com/bitnami-labs/flagenv` to operate on a `FlagSetVisitor` interface (just `Visit` and `VisitAll`, I think) to avoid the `pflagenv` variant fork?

Reply:
Pflag doesn't implement the same visit interface because it reimplements the Flag type.

Perhaps we can write an adapter.